modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheLifecycleAwareSelfTest.java

```diff
@@ -175,21 +171,6 @@ public TestAffinityFunction() {
         }
     }
 
-    /**
-     */
-    public static class TestEvictionPolicy extends TestLifecycleAware implements EvictionPolicy, Serializable {
```

Comment:
Why we removed eviction policy from testing?

Reply:
Because this behavhior exists for at least 5 years, test failed and nobody cares.

I think we can safely keep existing behaviour and modify test accordingly.